.github/workflows/rebase-release-proposal.yml

```diff
@@ -0,0 +1,86 @@
+name: Rebase release proposal
+
+on:
+  workflow_dispatch:
```

Comment:
Sometimes happen that the release proposal branch is green and have approval, but we are waiting because we want more approvals or some release notes updates.
I think that this should be manual for now, because once that this is merged, we should click on "publish npm" button and create a release in GH.

Reply:
I still think this could be automated even if reviews are required so that it would wait for the reviews, but that may be out of scope of this PR and could be re-evaluated later after we have evaluated the manual steps for a little while.